.github/workflows/tests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - "*"
+
+  schedule:
+    - cron: "59 23 * * 3"
+
+jobs:
+  Linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v4
+      - uses: pre-commit/action@v3.0.0
```

Comment:
it might be better to replace this with `pre-commit.ci`? The action is not actively maintained...